cmd/ddev/cmd/auth-ssh.go

```diff
@@ -45,13 +46,29 @@ var AuthSSHCommand = &cobra.Command{
 		}
 		fi, err := os.Stat(sshKeyPath)
 		if os.IsNotExist(err) {
-			util.Failed("The SSH key directory %s was not found", sshKeyPath)
+			util.Failed("The SSH key path %s was not found", sshKeyPath)
 		}
 		if err != nil {
-			util.Failed("Failed to check status of SSH key directory %s: %v", sshKeyPath, err)
+			util.Failed("Failed to check status of SSH key path %s: %v", sshKeyPath, err)
 		}
+
+		var paths []string
+		var files []string
 		if !fi.IsDir() {
-			util.Failed("The SSH key directory (%s) must be a directory", sshKeyPath)
+			files = append(files, sshKeyPath)
+		} else {
+			files, err = fileutil.ListFilesInDirFullPath(sshKeyPath, true)
+			if err != nil {
+				util.Failed("Failed to list files in %s: %v", sshKeyPath, err)
+			}
+		}
+		// Get real paths to key files in case they are symlinks
+		for _, file := range files {
+			realPath, err := filepath.EvalSymlinks(file)
```

Comment:
If they're symlinks, they'll have to be linked to something shared via docker provider, right?

Reply:
It's ugly. You can just put a key in /tmp and try it out.

Reply:
The error message should be more clear now.